modules/nextflow/src/test/groovy/nextflow/executor/FluxExecutorTest.groovy

```diff
@@ -89,6 +89,12 @@ class FluxExecutorTest extends Specification {
         task.config.clusterOptions = '--tasks-per-node=4 --cpus-per-node=4'
         then:
         executor.getSubmitCommandLine(task, Paths.get('/some/path/job.sh')) == ['flux', 'mini', 'submit', '--setattr=cwd=/work/path', '--job-name="nf-my_task"', '--output=/work/path/.command.log', '--time-limit=60', '--tasks-per-node=4', '--cpus-per-node=4', '/bin/bash', 'job.sh']
+
+        when:
+        task.config = new TaskConfig()
+        task.config.terminalOutput = true
```

Comment:
```suggestion
        task.config.flux.terminalOutput = true
```

Reply:
Oh I understand! So I’m allowed to define arbitrary variables under an “on the fly” flux namespace in the config - and navigate will allow me to do that (without a null error if it’s not defined). Gotcha! Sorry I missed this detail in your description above - it makes perfect sense. I’ll update the PR.